packages/ag-charts-community/src/util/color.ts

```diff
@@ -466,8 +466,7 @@ export class Color implements IColor {
      * CSS Color Module Level 4:
      * https://drafts.csswg.org/css-color/#named-colors
      */
-    private static readonly nameToHex: Record<string, string> = {
-        // @ts-expect-error
+    private static readonly nameToHex: Record<string, string | null> = {
         __proto__: null,
```

Comment:
nit: is `__proto__` really needed here?

Reply:
Discussed with @jacobp100 and switched to a `Map`.